blog/2025-02-19-virtual-hosted-urls/index.mdx

```diff
@@ -0,0 +1,154 @@
+---
+slug: virtual-hosted-urls
+title: "Moving to Virtual Hosted URLs"
+description: >
+  We’re transitioning to virtual hosted style URLs for all new buckets created
+  after February 19, 2025.
+keywords: [object storage, blob storage, s3]
+authors: [ks]
+tags: [s3]
+---
+
+import InlineCta from "@site/src/components/InlineCta";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+We’re transitioning to virtual hosted style URLs for all new buckets created
+after February 19, 2025. For new buckets, we will stop supporting path style
+URLs. Buckets created before February 19, 2025 will continue to work with either
+path style or virtual host style URLs.
+
+The path style URL looks like this:
+`https://fly.storage.tigris.dev/tigris-example/bar.txt`
+
+The virtual host style URL looks like this:
+`https://tigris-example.fly.storage.tigris.dev/bar.txt`
+
+With the path style URL, the subdomain is always `fly.storage.tigris.dev`. By
+moving to virtual host style URLs, the subdomain is specific to the bucket. This
+additional specificity allows us to make some key improvements for security and
+scalability.
+
+## Why make this change now?
+
+Recently some ISPs blocked the Tigris subdomain after
+[malicious content was briefly shared](https://status.tigris.dev/incidents/01JKCBMAVT1W09PGVEAB4XCWMZ)
+using our platform. Though we removed the malicious content, the subdomain was
+the common denominator across several reports and added to blocklist maintained
+by security vendors. This block of our domain resulted in failed downloads on
+several ISPs with unclear error messages. Either the DNS resolved to another IP
+not owned by Tigris, or there were connection errors that implied a network
+issue. We’re sure this was frustrating for folks to debug.
+
+We have been working with the security vendors to remove our domain from their
+blocklists. However, the long term solution is to move to virtual hosted style
+URLs so that the subdomains are no longer the common denominator when
+identifying content.
+
+## How does this impact your code?
+
```

Comment:
I think we don't have to mention custom domain in this blog post. We are not forcing users to use custom domain names with public buckets?